modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/TransactionStateResolver.java

```diff
@@ -129,6 +129,18 @@ public void start() {
         });
     }
 
+    /**
+     * Get consistent id by node id.
+     *
+     * @param id Node inconsistent id.
+     * @return Node consistent id.
+     */
+    public @Nullable String consistentId(String id) {
```

Comment:
its definitely not a task for tx state resolver, to resolve the node names by their ids